### DISCUSSION TITLE
Custom raw queries support

### DISCUSSION THREAD

**Discussion by akbar-idwise** [tailwindlabs/tailwindcss#11291] [Help] [answered]
Is it possible to use custom raw queries the way we can use with tailwind?

e.g
```
extend: {
  containers: {
    "xxs": { raw: "(max-height: 600px)" }
  }
}
```

Right now with svelte preprocessor we get this error:

`[!] (plugin svelte) TypeError: value.match is not a function`

**Comment by wongjn** [comment, marked as answer]
It is not possible to use custom raw queries for the `containers` the same way we can use raw queries in `screens` configuration. This is because `min-width` is baked into the `containers` rules:

https://github.com/tailwindlabs/tailwindcss-container-queries/blob/e53d76e6c5fd45005e30029417feb45683f0de2c/src/index.ts#L37

**Reply by akbar-idwise** [reply]
this is true for v0.1.1 of [tailwindcss-container-queries](https://github.com/tailwindlabs/tailwindcss-container-queries)
